content/en/docs/tasks/run-application/scale-stateful-set.md

```diff
@@ -13,29 +13,32 @@ weight: 50
 ---
 
 {{% capture overview %}}
-This page shows how to scale a StatefulSet.
+This task shows how to scale a StatefulSet. Scaling a StatefulSet refers to increasing or decreasing the number of replicas.
 {{% /capture %}}
 
 {{% capture prerequisites %}}
 
 * StatefulSets are only available in Kubernetes version 1.5 or later.
-* **Not all stateful applications scale nicely.** You need to understand your StatefulSets well before continuing. If you're unsure, remember that it might not be safe to scale your StatefulSets.
-* You should perform scaling only when you're sure that your stateful application
-  cluster is completely healthy.
+  To check your version of Kubernetes, run `kubectl version`.
+
+{{% caution %}}
```

Comment:
What value does this add? It provides only a little bit of offset in the page presentation. Recommend removing (the caution tag, not the content :-) )

Reply:
Done